spec/query_objects/filtered_articles_query_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe FilteredArticlesQuery do
+  subject(:all) { query.all }
+
+  let(:query) { described_class.new(relation, filter_params) }
+  let(:relation) { Article.all }
+
+  context "with no params" do
+    let(:filter_params) { {} }
+
+    it { is_expected.to match_array(relation) }
+  end
+
+  describe ".by_search" do
```

Comment:
because it's instance method
```suggestion
  describe "#by_search" do
```